mcp-server/Makefile

```
@@ -0,0 +1,102 @@
+# Licensed to Apache Software Foundation (ASF) under one or more contributor
+# license agreements. See the NOTICE file distributed with
+# this work for additional information regarding copyright
+# ownership. Apache Software Foundation (ASF) licenses this file to you under
+# the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+NAME := mcp-server
+DIST_INDEX := dist/index.js
+
+IMG_NAME := skywalking-banyandb-mcp-server
```

```suggestion
IMG_NAME := skywalking-banyandb-mcp
```

I think we should use this? To keep GitHub and DockerHub image repo name consistent?

---

@hanahmily Do we set the docker repo in the root? This is just IMG NAME, but we also need target URL.